src/cli/app.rs

```diff
@@ -134,6 +137,17 @@ pub fn create_app(
                       a root directory where output images will be copied, using a mirrored directory structure")
         )
 
+        // config for glob/batch mode
+        .arg(Arg::with_name(ARG_GLOB_SKIP_UNSUPPORTED_EXTENSIONS)
+            .long("skip-unsupported-extensions")
+            .takes_value(true)
```

Comment:
Alternatively we could make it a flag, but since it's likely (I assume) most people would like to skip the unsupported extensions by default, then we should reverse the name and behaviour of the flag, i.e. instead of `skip-unsupported-extension` use `allow-unsupported-extension`. On the other hand, having to provide an explicit choice with a default value also sounds good to me (I'm not sure sure yet what I would stylistically prefer). I wonder whether having an implicit default/explicit opt-out would be cleaner when having a flag, than having an explicit choice, with a default value set.

So far I didn't use explicit boolean values, only flags to represent boolean choices. So choosing  the flag version might be slightly  more consistent. On the other hand, having both opt-in and opt-out flags might be confusing from a user perspective.

Maybe we should also look at other cli apps and see what they do. If I recall correctly, `ripgrep` for example uses `--no-X` flags for opt-outs en `--flag` for opt-in.
Or perhaps we can simply try it out by merging this PR and determining whether we like this choice or should switch to using a flag. I will give it some more thought. If you have a particular opinion, feel free to share! 😄

Reply:
I've changed it to a flag named `--no-skip-unsupported-extensions`, in line with your explanation.

There is one place where explicit boolean values are used, namely for the modifier `PreserveAspectRatio`, but I can see that one may be more "special" since the image operations and related options seem to mirror the 'script' version counterparts :).